static/styles/night_mode.css

```diff
@@ -10,6 +10,27 @@ body.night-mode {
         color: inherit;
     }
 
+    .progressive-table-wrapper {
+        position: relative;
+        max-height: calc(95vh - 220px);
```

Comment:
We don't want to use `vh` in our code. It doesn't scale equally with browser zoom which is used quite frequently by Zulip users.

Reply:
Try using `%` values.

Reply:
> Try using `%` values.

@amanagr I tried using % values and tried playing around it but can't find the fix for it. Some more help from your side will be appreciable.

Reply:
@amanagr But isn't the basic meaning of `vh` is that it represents `%` of viewport height and zooming in/out doesn't affect the viewport size? So, I guess it should scale properly.

Reply:
Exactly, when zooming viewport size doesn't change but the size of all other elements change make them blow out of proportion. So, either define everything in terms of viewport or use units that depend on document size.

Reply:
I would suggest we just fix the `max-height: 800px`. If the height available height is less than that, it will just assume the overlay height. I don't think there will be any problem using that.

Reply:
I tried with `max-height: 800px` and its behaviour is looking like this. Is it normal behaviour?
![max-height-problem](https://user-images.githubusercontent.com/59444243/107123258-86d54500-68c2-11eb-9e60-696e2056a806.gif)